docs/content/installation/upgrade.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Upgrade"
+date:
+draft: false
+weight: 50
+---
+
+# Overview
+
+Upgrading to a new version of PGO is typically as simple as following the various installation
+guides defined within the PGO documentation:
+
+- [PGO Kustomize Install]({{< relref "./kustomize.md" >}})
+- [PGO Helm Install]({{< relref "./helm.md" >}})
+
+However, when upgrading to or from certain versions of PGO, extra steps may be required in order
+to ensure a clean and successful upgrade.  This page will therefore document any additional
+steps that must be completed when upgrading PGO.
+
+## Upgrading from PGO 5.0.0 Using Kustomize
+
+Starting with PGO 5.0.1, both the Deployment and ServiceAccount created when installing PGO via
+the installers in the
+[Postgres Operator examples repository](https://github.com/CrunchyData/postgres-operator-examples)
+have been renamed from `postgres-operator` to `pgo`.  As a result of this change, if using
+Kustomize to install PGO and upgrading from PGO 5.0.0, the following step must be completed prior
+to upgrading.  This will ensure multiple versions of PGO are not installed and running concurrently
+within your Kubernetes environment.
+
+Prior to upgrading PGO, first manually delete the PGO 5.0.0 `postgres-operator` Deployment and
+ServiceAccount:
+
+```bash
+kubectl -n postgres-operator delete deployment,serviceaccount postgres-operator
+```
+
+Then, once both the Deployment and ServiceAccount have been deleted, proceed with upgrading PGO
+by applying the new version of the Kustomize installer:
+
+```bash
+kubectl -n postgres-operator apply -k kustomize/install/bases
```

Comment:
❓ I forget, does `-n` work here or does `apply -k` always use what's in the Kustomization file?

Our [install docs](https://github.com/CrunchyData/postgres-operator/blob/74562db/docs/content/installation/kustomize.md#configuration) say to change the contents of the file. I suspect we should omit the `-n` flag here to be consistent.

Reply:
Good call - while `-n` can be specified with `apply -k`, it make sense to remove it here as we have on the Kustomize install page.  I therefore went ahead and removed `-n` for this command/example.